trunk/SUAVE/Analyses/Planets/Planet.py

```diff
@@ -1,52 +1,52 @@
-## @ingroup Analyses-Planets
-# Planet.py
-#
-# Created:  
-# Modified: Feb 2016, Andrew Wendorff
-
-# ----------------------------------------------------------------------
-#  Imports
-# ----------------------------------------------------------------------
-
-from SUAVE.Core import Data
-from SUAVE.Analyses import Analysis
-
-# ----------------------------------------------------------------------
-#  Analysis
-# ----------------------------------------------------------------------
-
-## @ingroup Analyses-Planets
-class Planet(Analysis):
-    """ SUAVE.Analyses.Planet()
-    """
-    
-    def __defaults__(self):
-        
-        """This sets the default values and methods for the analysis.
-    
-            Assumptions:
-            Planet is Earth.
-    
-            Source:
-            N/A
-    
-            Inputs:
-            None
-    
-            Outputs:
-            None
-    
-            Properties Used:
-            N/A
-            """                  
-        
-        
-        self.tag    = 'planet'
-        self.features = Data()
-        self.settings = Data()
-        
-        from SUAVE.Attributes.Planets.Earth import Earth
-        self.features = Earth()
-        
-        
+## @ingroup Analyses-Planets
```

Comment:
Why is this showing as different?

Reply:
For some reason it had three extra lines of whitespace at the bottom. They've been deleted.